activities.js

```diff
@@ -4,7 +4,7 @@ var os = require("os");
 
 var async = require("async");
 
-var swfr = require("./index");
+var worker = require("./index").Worker;
```

Comment:
Internally: `ActivityWorker` and `DeciderWorker` (the `stream.Writable` classes)

Externally: `activity` and `decider`